src/components/Checkout/AddressForm.jsx

```diff
@@ -14,12 +13,12 @@ export default function AddressForm({ formValues, onFormChange }) {
 
   return (
     <React.Fragment>
-      <Typography variant="h6" gutterBottom>
+      <Text variant="h4" gutterBottom>
```

Comment:
Here we changed from h6 to h4. Should it be the case?

Reply:
This happened because the scale I used is different from the MUI scale. I tried do keep de font size similar to the HTML default font size for each element.